src/main/java/training/supportbank/Transaction.java

```diff
@@ -0,0 +1,20 @@
+package training.supportbank;
+
+import java.math.BigDecimal;
+import java.util.Date;
```

Comment:
Java quirk: use `java.time.LocalDate` rather than `java.util.Date`.

`java.util.Date` is [old and buggy](https://stackoverflow.com/a/1969651/679924), and was replaced in Java 8 by the much better `java.time` package. That's extremely non-obvious if you're new to Java, though, so I'm sorry for not spotting it in the previous review!